chapters/cpp_for_opencl.md

```diff
@@ -1,8 +1,8 @@
 # C++ for OpenCL
 
-The OpenCL working group has transitioned from the original OpenCL C++ kernel language first defined in OpenCL 2.1 to the community developed  [C++ for OpenCL](https://github.com/KhronosGroup/Khronosdotorg/blob/master/api/opencl/assets/CXX_for_OpenCL.pdf) open source front end compiler that provides improved features and compatibility with OpenCL C.
+The OpenCL working group has transitioned from the original OpenCL C++ kernel language first defined in OpenCL 2.1 to the community developed [C++ for OpenCL](https://www.iwocl.org/wp-content/uploads/iwocl-syclcon-2020-stulova-13-slides.pdf) kernel language that provides improved features and compatibility with OpenCL C.
 
-C++ for OpenCL is supported by [Clang](https://clang.llvm.org/docs/UsersManual.html#c-for-opencl) and uses the [LLVM](https://llvm.org/) compiler infrastructure. Its implementation in Clang can be tracked via the [OpenCL Support Page](https://clang.llvm.org/docs/OpenCLSupport.html). It enables developers to use most C++17 features in OpenCL kernels and generates code, through offline compilation, in the SPIR-V intermediate representation that is ingested by an increasing number of OpenCL implementations.
+C++ for OpenCL enables developers to use most C++17 features in a kernel code while keeping familiar OpenCL constructs, syntax, and sematic from OpenCL C. This facilitates smooth transition to new C++ features in existing OpenCL applications and does not require changing familiar development flow or tools. The main design goal of C++ for OpenCL is to reapply OpenCL-specific concepts to C++ in the same way as OpenCL C does. Aside from minor exceptions OpenCL C is a valid subset of C++ for OpenCL. Overall, the kernel code written in C++ for OpenCL looks just like code written in OpenCL C with some extra C++ features available for convenience. 
```

Comment:
```suggestion
C++ for OpenCL enables developers to use most C++17 features in kernel code while keeping familiar OpenCL constructs, syntax, and semantics from OpenCL C. This facilitates a smooth transition to new C++ features in existing OpenCL applications and does not require changing familiar development flows or tools. The main design goal of C++ for OpenCL is to reapply OpenCL-specific concepts to C++ in the same way as OpenCL C does. Aside from minor exceptions OpenCL C is a valid subset of C++ for OpenCL. Overall, the kernel code written in C++ for OpenCL looks just like code written in OpenCL C with some extra C++ features available for convenience. 
```

This sentence is confusing:

> The main design goal of C++ for OpenCL is to reapply OpenCL-specific concepts to C++ in the same way as OpenCL C does

Did you intend to say "... as OpenCL C++ does"?

Reply:
No I mean to say in the same way as OpenCL C applies to OpenCL concepts to C...